### PR TITLE
fix PULLUP(s)

### PR DESCRIPTION
**Pull request by nerdCopter**
- fixes compiling of these targets
- unsure if they should also have `#define USE_I2C_PULLUP`

### CODE REVIEW

File: configs/ALIENFLIGHTF4/config.h [hunk 1]
```diff
@@ -120,4 +120,4 @@
 #define GYRO_1_SPI_INSTANCE SPI1
 #define GYRO_1_ALIGN CW270_DEG
 #define GYRO_1_ALIGN_YAW 2700
-#define I2C1_PULLUP ON
+//#define I2C1_PULLUP ON
```
Comment (haslinghuis) [inline configs/ALIENFLIGHTF4/config.h:123]: This needs to be
```suggestion
#define USE_I2C_PULLUP
```

**Comment by ledvinap** [comment]
It seems that `USE_I2C_PULLUP` is used to enable pullups (on all ports). https://github.com/betaflight/betaflight/blob/master/src/main/target/common_defaults_post.h#L42-L52

**Comment by haslinghuis** [comment]
All targets only use a single I2C port definition. If we want to have more control we need to change firmware instead.

**Comment by nerdCopter** [comment]
whatever ya'll want. let me know how to modify this PR.

**Review by haslinghuis — APPROVED** [review]
If we fix it in firmware it will apply to firmware 4.5.2 / 4.6 only.

**Comment by ot0tot** [comment]
Is this needed? The internal I2C pullups are very weak, I'm not sure enabling them does anything. There should be external pullups on the FC for I2C.

**Review by blckmn — APPROVED** [review]

**Pull request merged**